system/thanos-seeds/templates/seed.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: "openstack.stable.sap.cc/v1"
 kind: OpenstackSeed
 metadata:
-  name: {{ include "prometheus.name" (list $name $root) }}-thanos
+  name: {{ include "swift.userName" (list $name $root) | trimPrefix "prometheus-" }}
```

Comment:
It looks like you want to drag in clusterType here with this outcome:

kubernetes-clusterType-thanos

We should add the clusterType directive here then too:
https://github.com/sapcc/helm-charts/blob/master/system/thanos-seeds/values.yaml#L16-L25C3